src/instamatic/camera/camera_serval.py

```diff
@@ -46,12 +67,32 @@ def get_image(self, exposure=None, binsize=None, **kwargs) -> np.ndarray:
         """
         if exposure is None:
             exposure = self.default_exposure
-        if not binsize:
-            binsize = self.default_binsize
-
+        if exposure < self.MIN_EXPOSURE:
+            logger.warning(f'{self.BAD_EXPOSURE_MSG}: {exposure}')
+            return self._get_image_null()
+        elif self.MIN_EXPOSURE <= exposure <= self.MAX_EXPOSURE:
+            return self._get_image_single(exposure, binsize, **kwargs)
+        else:  # if exposure > self.MAX_EXPOSURE
+            logger.warning(f'{self.BAD_EXPOSURE_MSG}: {exposure}')
+            n_triggers = math.ceil(exposure / self.MAX_EXPOSURE)
+            exposure1 = (exposure + self.dead_time) / n_triggers - self.dead_time
+            arrays = self.get_movie(n_triggers, exposure1, binsize, **kwargs)
+            array_sum = sum(arrays, np.zeros_like(arrays[0]))
+            scaling_factor = exposure / exposure1 * n_triggers  # account for dead time
+            return (array_sum * scaling_factor).astype(array_sum.dtype)
```

Comment:
Consider moving this bit to its own function to be in line with the other 2 options:

```suggestion
            logger.warning(f'{self.BAD_EXPOSURE_MSG}: {exposure}')
            return _stacked_image(exposure, binsize, **kwargs)
```

Reply:
@stefsmeets Ok, this took some mental gymnastics but I have re-structured this part of the code again so now it not only looks better and has less repetition but also properly handles movies:

- All `get_image` and `get_movie` are redirected into a new `_get_image`
- `get_movie` properly return `[]` if requested `n_frames=0`;
- `get_image` / `get_movie` with exposure < lower limit now return `np.zeros` / `[np.zeros]`;
- `get_image` / `get_movie` with good exposure pass through 1 new if but work exactly as before;
- `get_image` with exposure > upper limit now works and returns a sum of movie arrays;
- `get_movie` with exposure > upper limit now also works and returns a list of partial sums of a more-trigger movie i.e. if requested exposures [12, 12], they will come as 2-sums of [6, 6, 6, 6].